src/core_plugins/kibana/public/dashboard/top_nav/options_popover.less

```diff
@@ -0,0 +1,7 @@
+.dashOptionsPopover {
+  height: 100%;
+
+  .euiPopover__anchor {
+    height: 100%;
+  }
+}
```

Comment:
cc @cchaos - here are some LESS changes that will probably go in before your LESS to SASS conversion.  Any objections?  Hopefully not to difficult to merge?

Reply:
Yeah that will be fine. I'll be able to handle the rebase.